extra/3-magic-8-ball.js

```diff
@@ -81,13 +130,13 @@ test("whole magic 8 ball sequence", () => {
 
   expect(consoleLogSpy).toHaveBeenCalledTimes(1);
   expect(consoleLogSpy).toHaveBeenLastCalledWith("The ball has shaken!");
-
-  expect(checkAnswer(answer)).toBeOneOf([
-    "very positive",
-    "positive",
-    "negative",
-    "very negative",
-  ]);
+  // I've found a problem with the code below,I've changed
```

Comment:
@pigos4 Thank you very much, I also had one issue even everything works well there was left one test not passed, have looked at who passed, I found your solution now is mine test all passed!